Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM node:lts
 WORKDIR /tmp
 COPY package* ./
```

Comment:
This line only copies files prefixed with `package` into the container image. It does not include `kodisync.js` hence the issue.

Reply:
This is done for caching purposes: the `npm ci` will be skipped/retrieved from cache if `package-lock.json` did not change.